Bank test fails due to changed error message

<!-- < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < < ☺ 
v                            ✰  Thanks for opening an issue! ✰    
v    Before smashing the submit button please review the template.
v    Please also ensure that this is not a duplicate issue :)  
☺ > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > > >  -->

<!--
IMPORTANT: Prior to opening a bug report, check if it affects one of the core modules
and if its elegible for a bug bounty on `SECURITY.md`. Bugs that are not submitted
through the appropriate channels won't receive any bounty.
 -->

## Summary of Bug

https://github.com/osmosis-labs/cosmos-sdk/commit/8cbb9d2864dc40255da129a48dec177a05c71864 caused the bank tests to fail, cause it changed the text of the error message

```
--- FAIL: TestMsgSendValidation (0.00s)
    msgs_test.go:52: 
        	Error Trace:	msgs_test.go:52
        	Error:      	Error message not equal:
        	            	expected: "Invalid recipient address (empty address string is not allowed): invalid address"
        	            	actual  : "Invalid recipient address :(decoding bech32 failed: invalid bech32 string length 0): invalid address"
        	Test:       	TestMsgSendValidation
FAIL
FAIL	github.com/cosmos/cosmos-sdk/x/bank/types	1.596s
```

## Version

8cbb9d2864dc40255da129a48dec177a05c71864

____

## For Admin Use

- [ ] Not duplicate issue
- [ ] Appropriate labels applied
- [ ] Appropriate contributors tagged
- [ ] Contributor assigned/self-assigned

--- a/x/bank/types/msgs_test.go
+++ b/x/bank/types/msgs_test.go
@@ -41,7 +41,7 @@ func TestMsgSendValidation(t *testing.T) {
 		{": invalid coins", NewMsgSend(addr1, addr2, atom0)},                   // non positive coin
 		{"123atom,0eth: invalid coins", NewMsgSend(addr1, addr2, atom123eth0)}, // non positive coin in multicoins
 		{"Invalid sender address (empty address string is not allowed): invalid address", NewMsgSend(addrEmpty, addr2, atom123)},
-		{"Invalid recipient address (empty address string is not allowed): invalid address", NewMsgSend(addr1, addrEmpty, atom123)},
+		{"Invalid recipient address :(decoding bech32 failed: invalid bech32 string length 0): invalid address", NewMsgSend(addr1, addrEmpty, atom123)},
 	}
 
 	for _, tc := range cases {

